src/client/client_base.cc

```diff
@@ -72,6 +72,11 @@ Status ClientBase::CreateData(const json& tree, ObjectID& id,
 }
 
 Status ClientBase::CreateMetaData(ObjectMeta& meta_data, ObjectID& id) {
+  if (this->instance_id_ == UnspecifiedInstanceID() - 1) {
```

Comment:
What the "this->instance_id_ == UnspecifiedInstanceID() - 1" means?

Reply:
It means whether the client is a RPC client. https://github.com/v6d-io/v6d/blob/main/src/client/rpc_client.cc#L131

Reply:
Use the `IsIPC()` and `IsRPC` mentioned in another comment.